test/test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+var fs = require('fs');
+var path = require('path');
+var rimraf = require('rimraf');
+var assert = require('assert');
+var assign = require('object-assign');
+var vinylFile = require('vinyl-file');
+
+var sass = require('../');
+var uniqueIntermediateDirectory = require('../utils').uniqueIntermediateDirectory;
+
+var defaultOptions = {
+	quiet: true,
+	unixNewlines: true // normalize compilation results on Windows systems
+};
+
+describe('compiles single file source', function () {
+	this.timeout(20000);
+	var files = [];
+
+	before(function(done) {
+		sass('source/file.scss', defaultOptions)
+		.on('data', function (data) {
+			files.push(data);
+		})
+		.on('end', done);
+	});
+
+	it('creates correct number of files', function () {
+		assert.equal(files.length, 1);
+	});
+
+	it('creates file at correct path', function () {
+		assert.equal(files[0].relative, 'file.css');
+	});
+
+	it('creates correct file contents', function () {
+		assert.equal(
+			files[0].contents.toString(),
+			new vinylFile.readSync('result/file.css').contents.toString()
```

Comment:
Not sure. I was thinking it would be a good idea to put them through the same process, but I suppose the end result is the same.

:+1:, will simplify.